src/components/GroupAndSearchDropdown/index.js

```diff
@@ -21,7 +21,7 @@ export const GroupAndSearchDropdown = ({
     const [searchString, setSearchString] = useState('')
 
     return (
-      <div className="options-dropdown">
+      <div id="dropdown" className="options-dropdown">
```

Comment:
We can remove `id`. As this component might me used multiple times and will result in duplicate `ids` in the DOM